### ISSUE TITLE
Can this model test iris

### ISSUE THREAD

**Issue by Strider20151** [iwyoo/ELM-tensorflow#4]
I use the iris dataset to test this model,but the accuracy is very low.Can you tell me the reason?
this is my test code:

from model import ELM
import tensorflow as tf
import pandas as pd
import numpy as np
#from sklearn.cross_validation import train_test_split
# Basic tf setting
tf.set_random_seed(2016)
sess = tf.Session()
# Loading the dataset
#iris数据集，测试数据和训练数据比例是3：2
dataset1 = pd.read_csv('uf2.csv')
dataset1 = pd.get_dummies(dataset1, columns=['species']) # One Hot Encoding
values1 = list(dataset1.columns.values)
y_train = dataset1[values1[-3:]]
y_train = np.array(y_train, dtype='float32')
#print(y)

X_train = dataset1[values1[0:-3]]

X_train = np.array(X_train, dtype='float32')
X_train-=np.min(X_train)
X_train/=np.max(X_train)

dataset2 = pd.read_csv('xapp2.csv')
dataset2 = pd.get_dummies(dataset2, columns=['species']) # One Hot Encoding
values2 = list(dataset2.columns.values)
y_test = dataset2[values2[-3:]]
y_test = np.array(y_test, dtype='float32')
#print(y)

X_test = dataset2[values2[0:-3]]

X_test = np.array(X_test, dtype='float32')
X_test-=np.min(X_test)
X_test/=np.max(X_test)

# Session
sess = tf.Session()

hidden_num=500
elm = ELM(sess,None, 4, hidden_num, 3)

# one-step feed-forward training
train_x = X_train
train_y=y_train

elm.feed(train_x, train_y)

# testing

elm.test(X_test,y_test)

**Comment by iwyoo** [comment]
나는 단순하게 논문을 구현하였을 뿐이기에, 논문에서 다루지 않은 데이터에 대해 성능을 보장할 수 없다. 또한 ELM은 아직 많은 실험을 통해 검증되지 않았다. 이 구현은 TensorFlow 같은 general한 deep learning 툴에서 ELM의 구현을 알아보기 위한 것일 뿐, 좋은 성능을 내기 위한 최적의 구현이 아닐 수 있다.
내 기억에는 `hidden_num`이 성능에 가장 크게 영향 미쳤던 것 같다. 만약 더 좋은 성능을 원한다면 `hidden_num`을 조절해보는 것이 좋을 것 같다.

**Comment by Strider20151** [comment]
ok，thank you very much.
